content/en/Platform Deep Dive/Scans/scans.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Scans"
+linkTitle: "Scans"
+weight: 15
+description: >
+  A scan represents a test run against a specific target
+---
+<style>
+  .image-box.borderless-image {
+    border: none;
+    border-radius: 0;
+    box-shadow: none;
+    margin: 0 0 20px 0;
+  }
+  .width400 {
+    width: 400px;
+  }
+</style>
+
+## Statuses
+
+A scan can have the following states:
+- Canceled
+- Canceling
+- Completed
+- Completed With Errors
+- Failed
+- Finishing Up
+- Paused
+- Pausing
+- Queued
+- Resuming
+- Running
+- Under Review
+
+To start a scan, you can either **start** it immediately or **schedule** it for a later point in time. This can even be on a [recurring basis].
+{{% image src="/deepdive/scans/scans/StartScan.png" alt="start scan button" class="borderless-image" %}}
```

Comment:
this one is clashing with the text a bit:
<img width="361" alt="Screenshot 2024-08-01 at 15 41 34" src="https://github.com/user-attachments/assets/3d13a401-962e-4fd2-8a66-bba75e544de1">